content/rooms/fragments/v2-state-res.md

```diff
@@ -36,10 +36,10 @@ The key-value pairs (*K*, *V*) across all state maps *S<sub>i</sub>* can be
 divided into two collections.
 If a given key *K* is present in every *S<sub>i</sub>* with the same value *V* 
 in each state map, then the pair (*K*, *V*) belongs to the *unconflicted state map*.
-Otherwise (*K*, *V*) belongs to the *conflicted state set*.
+Otherwise, *V* belongs to the *conflicted state set*.
 
 Note that the unconflicted state map only has one event for each key *K*,
-whereas the conflicted state set may associate multiple events to the same key.
+whereas the conflicted state set may contain multiple events with the same key.
```

Comment:
Perhaps "may contain multiple events associated to the same key"?

Reply:
> except it's not associated with a key

Agreed, but I didn't like the phrasing which suggested that events "have" a key. I think it's probably fine as is though.